f5-openstack-agent-dist/Docker/ubuntu/install_test/fetch_and_install_deps.py

```diff
@@ -1,140 +1,232 @@
 #!/usr/bin/python
 
+import glob
 import os
 import re
 import subprocess
 import sys
 
-f5_sdk_version_pattern = re.compile("^\s*Depends:\s+(?:.*)python-f5-sdk\s+\(=\s*(.*)\)(?:.*)$")
-f5_icr_version_pattern = re.compile("^\s*Depends:\s+(?:.*)python-f5-icontrol-rest\s+\(=\s*(.*)\)(?:.*)$")
+from collections import deque, namedtuple
+
+f5_sdk_rest_pattern = re.compile("^f5-sdk\s*=\s*(\d+\.\d+\.\d+)$")
+f5_sdk_rest_pattern = re.compile(
```

Comment:
Why was this redefined?  I am not sure that the f5-sdk-rest pattern will match what you want it to.